cmd/support_archive/logs.go

```diff
@@ -38,6 +39,12 @@ func newLogCollector(context context.Context, log logd.Logger, supportArchive ar
 }
 
 func (collector logCollector) Do() error {
+	if !installconfig.GetModules().Supportability {
```

Comment:
do we really want to skip the whole support archive if only extensions can't be collected, even in the case when it's not even used?

Reply:
we skip `stdout(s)`  and `*.log file(s)` only if `supportability` is disabled